tensorflow/python/layers/utils_test.py

```diff
@@ -18,9 +18,10 @@
 from __future__ import division
 from __future__ import print_function
 
-from tensorflow.python.layers import utils
 from tensorflow.python.platform import test
 
+from tensorflow.python.layers import utils
```

Comment:
This one the original order was okay?

Reply:
Yes. I have undone the import order change here.

Reply:
Thanks!